folly/Traits.h

```diff
@@ -384,10 +384,24 @@ struct Bools {
   }
 };
 
+template <bool... Bs>
+struct AllFalse : std::is_same<Bools<Bs...>, Bools<(Bs && false)...>> {};
+
+template <bool... Bs>
+struct AllTrue : std::is_same<Bools<Bs...>, Bools<(Bs || true)...>> {};
```

Comment:
I don't think we need another name for `StrictConjunction`. If we do, they should be called `AllOf`, `AnyOf`, and `NoneOf`, like the standard algorithms. But I don't see a need yet. I prefer to keep the API small and simple. `[Strict]Conjunction` and `[Strict]Disjunction` are all we need.